{
  "_from": "error-class@^2.0.1",
  "_id": "error-class@2.0.1",
  "_inBundle": false,
  "_integrity": "sha1-ZZpsXPOMlp2iobGwmodyWucTguc=",
  "_location": "/error-class",
  "_phantomChildren": {},
  "_requested": {
    "type": "range",
    "registry": true,
    "raw": "error-class@^2.0.1",
    "name": "error-class",
    "escapedName": "error-class",
    "rawSpec": "^2.0.1",
    "saveSpec": null,
    "fetchSpec": "^2.0.1"
  },
  "_requiredBy": [
    "/fortune"
  ],
  "_resolved": "https://registry.npmjs.org/error-class/-/error-class-2.0.1.tgz",
  "_shasum": "659a6c5cf38c969da2a1b1b09a87725ae71382e7",
  "_spec": "error-class@^2.0.1",
  "_where": "/Users/james.traver/ui/sample-json-api/node_modules/fortune",
  "author": {
    "email": "0x8890@airmail.cc",
    "url": "http://0x8890.com"
  },
  "bugs": {
    "url": "https://github.com/0x8890/error-class/issues"
  },
  "bundleDependencies": false,
  "deprecated": false,
  "description": "Typed errors that closely emulate the native Error class.",
  "devDependencies": {
    "eslint": "^2.11.1",
    "eslint-config-boss": "^1.0.3",
    "tapdance": "^4.1.1"
  },
  "engines": {
    "node": ">=4.2.0"
  },
  "eslintConfig": {
    "extends": "boss/es5",
    "rules": {
      "no-new-func": 0,
      "strict": 0
    }
  },
  "files": [
    "index.js",
    "LICENSE"
  ],
  "homepage": "https://github.com/0x8890/error-class#readme",
  "keywords": [
    "error",
    "type",
    "typed",
    "custom",
    "native"
  ],
  "license": "MIT",
  "name": "error-class",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/0x8890/error-class.git"
  },
  "scripts": {
    "lint": "eslint .",
    "test": "npm run lint && node test"
  },
  "version": "2.0.1"
}
